images/dockerregistry/Dockerfile

```diff
@@ -9,6 +9,9 @@ FROM openshift/origin-base
 RUN yum install -y tree findutils epel-release && \
     yum clean all
 
+# The registry doesn't require a privileged user.
+USER 1001
```

Comment:
Right workaround exists, no change necessary for 1.1, can deliver in 1.1.1

On Nov 10, 2015, at 8:30 AM, Paul Weil <notifications@github.com> wrote:

In images/dockerregistry/Dockerfile
<https://github.com/openshift/origin/pull/5805#discussion_r44428676>:

> @@ -9,6 +9,9 @@ FROM openshift/origin-base
>  RUN yum install -y tree findutils epel-release && \
>      yum clean all
>
> +# The registry doesn't require a privileged user.
> +USER 1001

Ok, will move. It's not a big issue, it can run with a higher or lower SCC.
If users would like to run with this particular SCC the workaround is just
to set a UID in the dc. Not a show stopper.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5805/files#r44428676>.